index.js

```diff
@@ -17,7 +17,9 @@ commander
   .option('-l, --logsql',
           'Log all sequelize queries and ddl')
   .option('-m, --message',
-          'Send message for each crawl stored to db (needs -s) to sqs queue');
+          'Send message for each crawl stored to db (needs -s) to sqs queue')
+  .option('-d, --delay <delay>',
+          'delay between crawls with forever mode, in seconds');
```

Comment:
It looks like we should add `parseFloat` or `parseInt` as a third argument to `option`
https://www.npmjs.com/package/commander